src/os/bluestore/BlueStore.h

```diff
@@ -3329,7 +3333,7 @@ class BlueStore : public ObjectStore,
     uint32_t op_flags = 0,
     uint64_t retry_count = 0);
 
-  void _do_read_and_pad(
+  bool _do_read_and_pad(
```

Comment:
missing a comment explaining what true & false in the retval mean

Reply:
done.